src/test/java/io/odh/test/unit/MockK8s.java

```diff
@@ -0,0 +1,123 @@
+package io.odh.test.unit;/*
+ * Copyright Strimzi and Broker QE authors.
```

Comment:
definitely no no broker QE... and no strimzi
follow same pattern skodjob authors...